api/camunda/camunda-openapi.yaml

```diff
@@ -1012,11 +1501,56 @@ components:
           format: int64
         tenantId:
           type: string
-    CamundaUserWithPasswordRequest:
+    AuthorizationAssignRequest:
+      type: object
+      properties:
+        ownerKey:
+          type: string
+        ownerType:
+          type: string
+          enum:
+            - USER
+            - ROLE
+            - GROUP
+        resourceKey:
+          type: string
+        resourceType:
+          type: string
+        permissions:
+          type: array
+          items:
+            type: string
+    UserWithPasswordRequest:
       type: "object"
       properties:
         password:
           type: "string"
+        username:
+          type: "string"
+        name:
+          type: "string"
+        email:
+          type: "string"
+    UserSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/UserFilterRequest"
+    UserFilterRequest:
+      type: object
+      properties:
+        username:
+          type: "string"
+        name:
+          type: "string"
+        email:
+          type: "string"
```

Comment:
change-at-the-source, descriptions, new:

These properties all need descriptions. We should follow up with the original author of this endpoint to add them.

Reply:
Nudged in https://github.com/camunda/camunda-docs/pull/4217#discussion_r1746211079